tmt/steps/report/polarion.py

```diff
@@ -204,6 +213,8 @@ def go(self, *, logger: Optional[tmt.log.Logger] = None) -> None:
         from tmt.export.polarion import find_polarion_case_ids, import_polarion
         import_polarion()
         from tmt.export.polarion import PolarionWorkItem
+
+        # TODO: Is this assert really necessary? Isn't this always true?
```

Comment:
@KwisatzHaderach is this assertion really necessary? Shouldn't we e.g. raise some ReportError exception if the `bool(PolarionWorkItem)` evaluates to `False`?

Reply:
probably not, I've added a lot of these to make typing happy, so if no linting tool complains then feel free to remove